test/runnable/gdb21158d.d

```diff
@@ -0,0 +1,30 @@
+/*
+REQUIRED_ARGS: -g
+PERMUTE_ARGS:
+GDB_SCRIPT:
+---
+b _Dmain
+r
+s
+s
+echo RESULT=
+info args
+---
+GDB_MATCH: RESULT=this =
+*/
+
+// We should obtain this = 0xYYY first = 4 second = 5
+// Unfortunately, there isn't a way to get only the second argument in GDB.
+// The first argument's memory address polutes the result.
```

Comment:
AFAIK, `GDB_MATCH` accepts a regex.

Reply:
If it does, I'll add this statement to test/README.md.

Reply:
It's already there:

> GDB_MATCH:   a regular expression describing the expected output of GDB_SCRIPT. The test
>                          will fail if it does not match the actual output.